packages/react-native-web/src/vendor/react-native/Animated/AnimatedEvent.js

```diff
@@ -4,7 +4,7 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  *
- * @noflow
+ * @flow
```

Comment:
Were the Animated files updated from RN or just typed the old ones?

Reply:
Typed the old ones I believe